### PR TITLE
Remove default value for Customer Group form

### PR DESCRIPTION
**Pull request by M0rgan01**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Remove default value for Customer Group form.
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | see #32583
| Fixed ticket?     | #32583
| Related PRs       | If theme, autoupgrade or other module change is needed to make this change work, provide a link to related PRs here.
| Sponsor company   | Your company or customer's name goes here (if applicable).

### CODE REVIEW

**Comment by nesrineabdmouleh** [comment]
Thank you @M0rgan01 :)
Nightly run => https://github.com/nesrineabdmouleh/testing_pr/actions/runs/4991874367

**Review by matks — APPROVED** [review]

File: src/PrestaShopBundle/Form/Admin/Sell/Customer/GroupType.php [hunk 1]
```diff
@@ -53,10 +53,8 @@ public function getParent(): string
     public function configureOptions(OptionsResolver $resolver): void
     {
         $resolver->setDefaults([
-            'placeholder' => null,
             'choices' => $this->groupByIdChoiceProvider->getChoices(),
             'choice_translation_domain' => false,
-            'autocomplete' => true,
```
Comment (kpodemski) [inline src/PrestaShopBundle/Form/Admin/Sell/Customer/GroupType.php:59]: @M0rgan01 you sure it is required to fix the issue? It adds select2 to the input if I recall correctly.

Comment (M0rgan01) [reply src/PrestaShopBundle/Form/Admin/Sell/Customer/GroupType.php:59]: Yes, It was not present before the regression, and the rendering is not good with

Capture d’écran du 2023-05-16 15-23-50

**Review by FabienPapet — APPROVED** [review]

**Review by kpodemski — APPROVED** [review]

**Review by florine2623 — APPROVED** [review]
Hello @M0rgan01 ,

Issue is fixed with this PR. No group is selected by default.

Tested the filter for Groups and other elements.
It is looking good.

It is QA ✅

**Comment by boherm** [comment]
Thank you @M0rgan01!

**Pull request merged**